bouncer.go

```diff
@@ -267,29 +309,52 @@ func handleNoStreamCache(bouncer *Bouncer, remoteIP string) error {
 		return fmt.Errorf("handleNoStreamCache:parseDuration %w", err)
 	}
 	if isLiveMode {
-		cache.SetDecision(remoteIP, true, int64(duration.Seconds()))
+		bouncer.cacheClient.SetDecision(remoteIP, true, int64(duration.Seconds()))
 	}
 	return fmt.Errorf("handleNoStreamCache:banned")
 }
 
+func getToken(bouncer *Bouncer) {
+	loginURL := url.URL{
+		Scheme: bouncer.crowdsecScheme,
+		Host:   bouncer.crowdsecHost,
+		Path:   crowdsecCapiLogin,
+	}
+	body, err := crowdsecQuery(bouncer, loginURL.String(), true)
+	if err != nil {
+		logger.Error(err.Error())
+	}
+	var login Login
+	err = json.Unmarshal(body, &login)
+	if err != nil {
+		logger.Error(fmt.Sprintf("getToken:parsingBody %s", err))
+		isCrowdsecStreamHealthy = false
+		return
+	}
+	if login.Code == 200 && len(login.Token) > 0 {
+		bouncer.crowdsecKey = login.Token
+	}
```

Comment:
Should add en else with an error log to help us debug when user provides bad credentials